### ISSUE TITLE
DockerHub release workflows broken

### ISSUE THREAD

**Issue by benfrancis** [WebThingsIO/registration_server#104]
DockerHub now requires a personal access token instead of a password

```
Error response from daemon: Get "https://registry-1.docker.io/v2/": unauthorized: your account must log in with a Personal Access Token (PAT) - learn more at docs.docker.com/go/access-tokens
Error: Command failed: docker_buildx/scripts/docker_login.sh *** *** 
```

**Comment by benfrancis** [comment]
@tim-hellhake I tried to fix this with https://github.com/WebThingsIO/registration_server/pull/105 by replacing password with a personal access token as we did with https://github.com/WebThingsIO/addon-builder/commit/13f94ebaa9416871329a753e9974a04b5864c5f0#diff-6e17565eb21d66eba4e808cded354e8dd1ec2f5c580ab6632c708acb2d2c509fR45

Unfortunately the workflow still fails with the error "Error: The parameter dockerPassword is missing". so it requires a bit more investigation. If the current GitHub action doesn't support personal access tokens we may have to switch to a newer workflow. See https://docs.docker.com/build/ci/github-actions/

I will try to take another look tomorrow but if you get a chance sooner then please do because I am expecting outgoing email to stop working for the webthings.io registration server at some point today.

**Comment by tim-hellhake** [comment]
@benfrancis it fails because there no secret called `DOCKER_HUB_PAT` present in this repository.
You can add it here: https://github.com/WebThingsIO/registration_server/settings/secrets/actions.
It's also possible to set the secret in the scope of the organisation: https://github.com/organizations/WebThingsIO/settings/secrets/actions

**Comment by benfrancis** [comment]
🤦 thanks! I've updated the username and personal access token secrets in the repository.

The release workflow is now failing with a new error:

`2.613 E: The repository 'http://deb.debian.org/debian buster-backports Release' does not have a Release file.`

I have proposed a temporary fix for this error in https://github.com/WebThingsIO/registration_server/pull/106

**Comment by benfrancis** [comment]
Fixed by #106
